MyApp/ViewController.swift

```diff
@@ -14,6 +14,9 @@ class ViewController: UIViewController {
         // Do any additional setup after loading the view.
     }
 
-
+    @IBAction func openAutoLayouts(_ sender: UIButton) {
+        performSegue(withIdentifier: "BasicAutoLayout", sender: self)
```

Comment:
move identifier ro constant file and it's first letter should be start with small case